thirdparty/faiss/faiss/utils/distances.h

```diff
@@ -422,6 +422,7 @@ void compute_PQ_dis_tables_dsub2(
  * @param y    database vectors, size ny * d
  * @param ids  result array ids
  * @param val  result array value
+ * @param data the pointer of memory for symmetric matrix data
```

Comment:
Please call this variable `tmp_buffer` in order to stress out its meaning. It is not a data, it is a temporary storage.

Reply:
Thanks for helping review, Alex! The change is done.